src/pages/Homepage.jsx

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
 import Booklist from '../components/Booklist';
 import Form from '../components/Form';
+import Footer from '../components/Footer';
 
 const Homepage = () => (
   <div>
```

Comment:
- [ ] I appreciate your dedication to the project! To ensure the best possible outcome, I highly recommend following the design guidelines closely. As you can observe, the components are enclosed within a div that features a box shadow effect in a specific color. For optimal results, kindly adhere to the design's instructions by incorporating the outer wrap div as depicted. Don't forget to include the necessary details and adjust the layout accordingly to match the design precisely.


| Your | Design |
| --- | --- |
|  ![image](https://github.com/AliRezaBmeDu/bookstore/assets/55840999/eda34fc0-d6e4-41bc-bbc4-f214dc0b18ae)| ![Bookstore CMS (1)](https://github.com/AliRezaBmeDu/bookstore/assets/55840999/c5ee8926-125e-4e00-be1c-ee31bcb35807)|

Your commitment to mirroring the design's structure will undoubtedly enhance the overall visual appeal of your project. Keep up the fantastic work! 🌟🎨